docs/en/your-profile-and-preferences/customize-membership-request-notifications.md

```diff
@@ -0,0 +1,40 @@
+---
+id: 584
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Customize membership request notifications
+
+!!! note ""
+    Currently, you can only customize membership request notifications using  Status mobile.
+
+If you require [manual approval][manage-community-join-requests] to join your community, you receive a notification for every member who wants to join your community.  
```

Comment:
- 'If manual request approval is required'?
- '<...> receive a notification every time someone requests to join your community'?